### ISSUE TITLE
Screen Reader: Workspace: 3-dot More button in Workspace container not focusable or operable

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77471]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS, open the app
2. Swipe right to focus the Workspaces tab and double-tap to activate it
3. Swipe right and try focusing the 3-dot button of any workspace container

## Expected Result:
Each workspace container should be focusable as a row, and the internal More (3-dot) button must be individually focusable and operable.
The More button should be right after the container in focus order.

## Actual Result:
The workspace container is treated as a single accessible element so the inner More button is not exposed to VoiceOver:
VoiceOver users are unaware of the button and cannot interact with it.

## Area issue was found in:
Workspace

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Use two separate accessibility elements inside the same visual workspace container.
1. Do not set accessible={true} on the parent container - This prevents iOS from focusing any child elements within the container.
2. Wrap the parent’s operable area in its own Pressable - This is what receives focus as the workspace button.
3. Keep the 3-dot button as a separate sibling element - VoiceOver sees it as a separate control.

Refer to:
https://reactnative.dev/docs/accessibility#accessible

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2428" alt="Image" src="https://github.com/user-attachments/assets/0810f729-5d5a-4db6-8f35-0735572d0e79" />

https://github.com/user-attachments/assets/5511c120-44b1-4aa5-9b47-6814fd784983

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7007079
**Issue reported by:** NA
**Slack conversation:** NA
